src/cow-react/modules/limitOrders/pure/Orders/OrderRow/EstimatedExecutionPrice.tsx

```diff
@@ -66,7 +76,7 @@ export function EstimatedExecutionPrice(props: EstimatedExecutionPriceProps) {
     ? amountDifference.multiply(MINUS_ONE_FRACTION)
     : amountDifference
   const orderExecutionStatus = calculateOrderExecutionStatus(percentageDifferenceInverted)
-  const feeWarning = percentageFee?.greaterThan(TEN_PERCENT)
+  const feeWarning = canShowWarning && percentageFee?.greaterThan(TEN_PERCENT)
```

Comment:
nit: I wonder if instead of this being calculated here, we should remove the prop `canShowWarning` and  add a `showUnlikelyToExecuteWarning` prop, so the component is dumber